Extension/src/LanguageServer/cppBuildTaskProvider.ts

```diff
@@ -185,9 +185,9 @@ export class CppBuildTaskProvider implements TaskProvider {
 
         const scope: TaskScope = TaskScope.Workspace;
         const task: CppBuildTask = new Task(definition, scope, definition.label, CppBuildTaskProvider.CppBuildSourceStr,
-            new CustomExecution(async (): Promise<Pseudoterminal> =>
+            new CustomExecution(async (resolvedDefinition: TaskDefinition): Promise<Pseudoterminal> =>
                 // When the task is executed, this callback will run. Here, we setup for running the task.
-                new CustomBuildTaskTerminal(resolvedcompilerPath, definition ? definition.args : [], definition ? definition.options : undefined)
+                new CustomBuildTaskTerminal(resolvedcompilerPath, resolvedDefinition ? resolvedDefinition.args : [], resolvedDefinition ? resolvedDefinition.options : undefined)
```

Comment:
Can it be `resolvedDefinition?.options` instead of `resolvedDefinition ? resolvedDefinition.options : undefined`?

Reply:
Sure

Reply:
The ? checks were actually unnecessary.